runtime/chainerx_util.h

```diff
@@ -43,5 +43,14 @@ bool IsFloat(chainerx::Dtype dtype);
 
 void BlitArray(const chainerx::Array& src, const chainerx::Array& dst);
 
+chainerx::Array MatMul(const chainerx::Array& a, const chainerx::Array& b);
```

Comment:
Could you rename this to something like BatchMatmul? When ChainerX adds MatMul, I think we will silently start using chainerx::MatMul instead of this function due to ADL (which I hate, btw), and we may forget to remove this function or worse overlook incompatibility between these two functions